src/Symfony/Component/Process/Tests/ProcessTest.php

```diff
@@ -1342,6 +1342,21 @@ public function testNonBlockingNorClearingIteratorOutput()
         $this->assertSame($expectedOutput, $output);
     }
 
+    public function testChainedProcesses()
+    {
+        $p1 = new Process(self::$phpBin.' -r '.escapeshellarg('fwrite(STDERR, 123); fwrite(STDOUT, 456);'));
```

Comment:
Was not aware of ``getProcess`` being usable only once per test. so fine with me